core/templates/pages/subtopic-viewer-page/navbar-breadcrumb/subtopic-viewer-navbar-breadcrumb.component.html

```diff
@@ -66,4 +66,10 @@
     color: white;
     font-size: 12px;
   }
+
+  @media only screen and (min-width: 769px) and (max-width: 970px) {
+ .oppia-subtopic-viewer-nav-labels-container {
```

Comment:
@mannika763 Please fix the indentation here so that it is 2 spaces more indented than the previous line. (See e.g. line 3592 of the oppia.css file above for an example.)